chain/indexer/integrated/processor/state.go

```diff
@@ -324,3 +385,197 @@ func (sp *StateProcessor) startActor(ctx context.Context, current, executed *typ
 	}()
 	return taskNames
 }
+
+type IndexerProcessors struct {
+	TipsetProcessors  map[string]TipSetProcessor
+	TipsetsProcessors map[string]TipSetsProcessor
+	ActorProcessors   map[string]ActorProcessor
+	ReportProcessors  map[string]ReportProcessor
+}
+
+func MakeProcessors(api tasks.DataSource, indexerTasks []string) (*IndexerProcessors, error) {
```

Comment:
was moved from `tipset.go`, mechanical change with tests for the method in `state_test.go`